rmw_opensplice_cpp/src/types.hpp

```diff
@@ -184,7 +184,6 @@ class OpenSplicePublisherListener : public DDS::PublisherListener
   virtual void on_publication_matched(
     DDS::DataWriter_ptr writer,
     const DDS::PublicationMatchedStatus & status);
-
```

Comment:
Removed new line to make code style more consistent with the rest of the file.

Reply:
Please, avoid unrelated changes.
In any case, adding the missing spaces in other places would be better.